vuepress/docs/v6.3.2/tutorials/create/ms/generate-microservices-and-micro-frontends.md

```diff
@@ -38,76 +39,85 @@ npm install -g generator-jhipster-entando@6.3.2
-
-You now have a Entando project including a Spring Boot microservice with database integration and React-based micro frontends.      
+
+5. You'll be presented with a series of prompts to add fields to your entity. These are listed below, with the base values for this tutorial in parentheses. Input what works best for you, and note that hitting "Enter" will select the default option.
+
+    - `Do you want to add a field to your entity?` (Yes)
+
+    - `What is the name of your field?` ("name")
+    - `What is the type of your field?` (String)
+    - `Do you want to add validation rules to your field?` (No)
+    - `Do you want to add a field to your entity?` (Yes)
+    - `What is the name of your field?` ("location")
+    - `What is the type of your field?` (String)
+    - `Do you want to add validation rules to your field?` (No)
+    - `Do you want to add a field to your entity?` (No)
+    - `Do you want to add a relationship to another entity?` (No)
+    - `Do you want to use separate service class for your business logic?` (Up to you)
+       - If yes:
+       - `Do you want to use a Data Transfer Object (DTO)?` (Up to you)
+       - `Do you want to add filtering?` (Up to you)
+    - `Is this entity read-only?` (Up to you)
+    - `Do you want pagination on your entity?` (Yes, with infinite scroll)
+
+- If you asked to be prompted about micro frontends when generating the project:  
+    - `Do you want to generate micro frontends?` (Up to you)
+
+- At this point the blueprint will generate controllers, repositories, services, and micro frontends for your entity.  
+    - `Overwrite src/main/resources/config/liquibase/master.xml?` When prompted with a conflict at this stage, enter `a` for "All." This will override existing files with the configuration changes needed for your new entity.
+
+You now have an Entando project, including a Spring Boot microservice with database integration and React-based micro frontends.      
 
 ### Project Structure
-   * ```/src/main/docker``` contains Docker files to help with local development environments
-   * ```/src/main/java``` and ```src/main/resources``` contain the microservice codebase and configuration
-   * ```/ui``` holds the React-based micro frontends. By default each entity gets an MFE for details, form, and table.
-   * ```/bundle``` is used to assemble the project code into an Entando bundle.
+   * ```/src/main/docker``` contains Docker files to help with local development environments.
+   * ```/src/main/java``` and ```src/main/resources``` contain the microservice codebase and configuration.
+   * ```/ui``` holds the React-based micro frontends. By default, each entity is allocated an MFE for details, form, and table.
+   * ```/bundle``` is used to assemble the project code into an Entando Bundle.
 
 ## Next Steps
 You now have a choice:
```

Comment:
this reads kind of fatalistic? Maybe 'Congratulations on ... then now you can choose between these ...'

Reply:
LOL it's the matrix all over again. i didn't care enough to change it? but yeah i'll rephrase to make it seem a little less dire